src/sidebar/Sidebar.jsx

```diff
@@ -35,67 +36,79 @@ export default function Sidebar(){
+                  </Link>
+                </OverlayTrigger>
+                <ul
+                  className={
+                    isDropdownActive ? "list-unstyled  collapse" : "list-unstyled"
+                  }
+                >
+                  <li className="dropdown-item">
+                    <Link to="/portfolio" className="sidebar-link">Portfolio</Link>
+                  </li>
+                  <li className="dropdown-item">
+                    <Link to="/personal-details" className="sidebar-link">Personal Details</Link>
+                  </li>
+                  <li className="dropdown-item">
+                    <Link to="/additional-info" className="sidebar-link">Additional Info</Link>
+                  </li>
+                  <li className="dropdown-item">
+                    <Link to="/personal-background" className="sidebar-link">Personal Background</Link>
+                  </li>
+                </ul>
+              </li>
+              <li className="list-item">
+                <Link to="/request-history" className="sidebar-link">
+                  <i className="fas fa-history icon-color"></i>
+                  <span className="sidebar-link-name">Request History</span>
+                </Link>
+              </li>
+              <li className="list-item">
+                <Link to="/organization-profile" className="sidebar-link">
+                  <i className="fas fa-sitemap icon-color"></i>
+                  <span className="sidebar-link-name">My Organization</span>
                 </Link>
-              </OverlayTrigger>
-              <ul
-                className={
-                  isDropdownActive ? "list-unstyled  collapse" : "list-unstyled"
-                }
-                id="homeSubmenu"
-              >
-                <li className="dropdown-item">
-                  <Link to="/portfolio">Portfolio</Link>
-                </li>
-                <li className="dropdown-item">
-                  <Link to="/personal-details">Personal Details</Link>
-                </li>
-                <li className="dropdown-item">
-                  <Link to="/additional-info">Additional Info</Link>
-                </li>
-                <li className="dropdown-item">
-                  <Link to="/personal-background">Personal Background</Link>
-                </li>
-              </ul>
-            </li>
-            <li className="list-item">
-              <i className="fas fa-history icon-color"></i>
-              <Link to="/request-history">Request History</Link>
-            </li>
-            <li className="list-item">
-              <i className="fas fa-sitemap icon-color"></i>
-              <Link to="/organization-profile">My Organization</Link>
-            </li>
-          </ul>
-        </nav>
+              </li>
+            </ul>
+          </nav>
+        </div>
       </div>
-    </div>
-  );
-};
+    );
+  }else{
```

Comment:
You can use ternary operator instead of if-else. Like `return isAuth? <component> : ""`